Projects/Features/SearchFeature/Sources/Reactors/SearchReactor.swift

```diff
@@ -57,14 +59,27 @@ final class SearchReactor: Reactor {
             newState.typingState = state
         case let .updateText(text):
             newState.text = text
+        case let .fetchRecentText(text):
+            break
         }
 
         return newState
     }
+
+    func transform(mutation: Observable<Mutation>) -> Observable<Mutation> {
+        let servicetypingState = DefaultSearchCommonService.shared.typingStatus
+            .map { Mutation.updateTypingState(state: $0) }
+
+        let text = DefaultSearchCommonService.shared.recentText.map { Mutation.updateText($0) }
```

Comment:
```suggestion
        let text = service.recentText.map { Mutation.updateText($0) }
```
이러면 Service를 주입받은 의미가 없는거같아요

Reply:
반영했습니다.